tests/doubleandfloat/printdf.jou

```diff
@@ -0,0 +1,7 @@
+declare printf(pattern: byte*, ...) -> int
+
+def main() -> int:
+    printf("%f\n", 114.514) # Normal
+    printf("%.5f\n", 3.1415926) # Format #1
+    printf("%.16f\n", 3.14159265358979323846264338320795) # Format max #2

```

Comment:
I think this doesn't use `float` anywhere. Everything here is `double`. Can you delete this test? It is already covered by `tests/should_succeed/add_sub_mul_div_mod.jou`.

Reply:
okay

Reply:
removed it